sources/install/package_ad.sh

```diff
@@ -1366,6 +1366,20 @@ function install_sccmwtf() {
     add-to-list "sccmwtf,https://github.com/xpn/sccmwtf,This code is designed for exploring SCCM in a lab."
 }
 
+function install_smbclientng() {
+    colorecho "Installing smbclientng"
+    git -C /opt/tools/ clone --depth 1 https://github.com/p0dalirius/smbclientng /opt/tools/smbclientng/
+    cd /opt/tools/smbclientng/ || exit
+    python3 -m venv --system-site-packages ./venv
+    source ./venv/bin/activate
+    pip3 install -r requirements.txt
+    deactivate
```

Comment:
You can use pipx to install the tool !

`pipx install git+https://github.com/p0dalirius/smbclient-ng`

Reply:
I used pip3 since other functions installing tools were doing this :)

I have changed to pipx in the latest commit